tests/test_array.py

```diff
@@ -366,9 +366,9 @@ def test_chunks_initialized(test_cls: type[Array] | type[AsyncArray[Any]]) -> No
         arr[region] = 1
 
         if test_cls == Array:
-            observed = sorted(chunks_initialized(arr))
+            observed = sorted(await chunks_initialized(arr))  # Why doesn't mypy error here?
```

Comment:
Oh good, this failed in CI: https://results.pre-commit.ci/run/github/48049137/1730299519.ojFfXsB_QvipnKxxQtQKgA

And it fails for me locally if I run it on `--all-files` (but it didn't fail when checking the diff, part of why I don't like putting mypy in pre-commit).

I'll get this fixed up. AFAICT, `chunks_initialized` isn't part of the public API so it's safe to make it async only.